async/retry.ts

```diff
@@ -12,15 +12,15 @@ export class RetryError extends Error {
 export interface RetryOptions {
   /** How much to backoff after each retry. This is `2` by default. */
   multiplier?: number;
-  /** The maximum milliseconds between retries. This is `60000` by default. */
+  /** The maximum milliseconds between retries. This is `60000` by default. `-1` indicates no maximum. */
```

Comment:
> `-1` indicates no maximum.

Why is this handling necessary? The users can pass `maxTimeout: Infinity` if they don't want maximum timeout

Reply:
I just added that to document the existing behavior as of the current version. I guess removing the behavior wouldn't be a breaking change as it's currently undocumented.